tools/ports/sdl2.py

```diff
@@ -82,11 +82,12 @@ def create():
     ports.create_lib(final, o_s)
     return final
 
-  return [shared.Cache.get(libname, create, what='port')]
+  return [shared.Cache.getlib(libname, create, what='port')]
 
 
 def clear(ports, settings, shared):
-  shared.Cache.erase_file(ports.get_lib_name('libSDL2'))
+  libname = ports.get_lib_name('libSDL2' + ('-mt' if settings.USE_PTHREADS else ''))
```

Comment:
Is the mt change separate from the main part of this PR? Or how is it connected?

Reply:
I think its separable, but an existing bug.  The name of the lib being erased was wrong in the pthread case.   I will split out.